### ISSUE TITLE
Custom message for format validation 

### ISSUE THREAD

**Issue by marcocanderle** [refile/refile#457]
Hello,

I'm using https://github.com/jbrien/refile-mongoid for my project. I need to know if there is any way of customizing the validation message returned when a file is attempted with an invalid format.

I have this on my model

```
attachment :file, type: :image
```

When I try to upload, say, a txt file, I get the "is invalid" message.

The question is, is there a way of customizing the validation message? I would like to have a validation message like: 

```
You are not allowed to upload "txt" files, allowed types: jpg, jpeg, gif, png
```

I tried replacing the corresponding locale entry but I don't know how to set it so that the allowed types and the uploaded file extension are included on the message.

Thanks!

**Comment by sobrinho** [comment]
You need to replace those I18n entries:

```yml
en:
  activerecord:
    errors:
      messages:
        too_large: "is too large"
        download_failed: "could not be downloaded"
        invalid_content_type: "has an invalid file format"
        invalid_extension: "has an invalid file format"
```

These are the defaults, as specified here: https://github.com/refile/refile/blob/3733ade8359cde398e3eb003f6fc97dd4be11ef7/config/locales/en.yml

Remember to change the language key if you have changed it in your application.

But if you need the allowed types or extensions, this isn't implemented yet.

I would be glad to accept a pull request implementing this.

**Comment by sobrinho** [comment]
Fixed by #564.
